.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build server library
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        configuration: [Debug]
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # Checkout your code
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 
+      # Use the action to generate a server library
+      - name: Generate ASPNET Core server
+        uses: openapi-generators/openapitools-generator-action@v1
+        with:
+          generator: aspnetcore
+          openapi-file: PxAPI-2.yml
+          config-file: aspnetcore-generator-config.yml
+          generator-tag: v6.5.0
+          command-args: -o .
+
+      - name: Check for staged changes
+        run: |
+          git diff --exit-code
```

Comment:
@likp this will detect if spec and code is out outof sync

Reply:
Nice. Should we removed the regeneration from the nuget workflow and just use the files already generated?

Reply:
Yes, I'll do it first thing tomorrow😀

Reply:
Ok, now it's done 👍